python/paddle/fluid/tests/book/high-level-api/recognize_digits/test_recognize_digits_conv.py

```diff
@@ -95,6 +96,9 @@ def event_handler(event):
             #         event.epoch + 1, float(avg_cost), float(acc)))
             #     if math.isnan(float(avg_cost)):
             #         sys.exit("got NaN loss, training failed.")
+        elif isinstance(event, fluid.EndStepEvent):
+            print("Step {0}, Epoch {1} Metrics {2}".format(
+                event.step, event.epoch, map(numpy.array, event.metrics)))
```

Comment:
Maybe we don't want to expose LoDTensor to the user. Is it possible to convert `event.metrics` implicitly to numpy.ndarray by PaddlePaddle?

Reply:
I am not sure how to represent `LoDTensor` as a ndarray. Tensor is a ndarray. However, LoDTensor is ndarray with sequence information.

There could be several solutions.

1. Cast LoDTensor as ndarray when `lod_level=0`. Otherwise, return LoDTensor. However, the element in metrics could be both `LoDTensor` and `ndarray`.
2. We use a sequence of ndarray to represent LoDTensor. It is slow and the element in metrics could be `ndarray`, `list[ndarray]` or `list[list[ndarray]]`.
3. Just return `LoDTensor`. The element type of metrics is `LoDTensor`. The behaviour of metrics is consistent.

Here the API always returns `LoDTensor` for simplicity.

Reply:
Thanks, let's use `LoDTensor` for simplicity for now.